src/components/Login/SignUpViews/SignUpView.tsx

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import { View, StyleSheet } from "react-native";
+
+const SignUpView = () => {
+  return <View></View>;
+};
+
+const styles = StyleSheet.create({});
+
+export default SignUpView;
```

Comment:
Why is there an empty component here?

Reply:
Regarding empty views, this is intentional as I wanted to get some of our additional pages laid out ready for me to add to the navigation. As we’re not on sprint 3 yet I have started work on the pages themselves yet though.